### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by victormc13**
### Latest Features:
---
**General requirements**
- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- HTML/CSS & JavaScript requirements
Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

**Project requirements**

I implemented the following interactions:

- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
In order to associate each project with the popup window details, you must refactor the project section:

- I have been to use a JavaScript array to store all of the information for all projects.
- For each project I have to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- I have was to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. 
- I implemented popups for both mobile and desktop screen sizes.

### CODE REVIEW

**Review by Strangeal — COMMENTED** [review]
Hi @victormc13 and @diarisdiakite, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have `eslint` setup in your linters. Kindly add it and make sure all tests are passing before submitting it for review. Here is a [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) to add it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by victormc13** [comment]
Hi @Strangeal, I think  already it is because if you see the files in the branch there it is the .eslintrc.json file and I have run all the commands to configure it.
image

**Review by amejid — COMMENTED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, where ESLint is not running. Kindly update the `linters.yml` file with [this one](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -585,20 +675,54 @@ textarea:focus {
...
+    border-radius: 10px;
+    padding: 1rem;
+    gap: 1rem;
+  }
+
+  .project-image {
+    height: 580px;
+    min-width: 100%;
+    background: url(icons/Snapshoot\ Portfolio\ desktop.svg) no-repeat center;
+  }
```
Comment (jerryowusu) [inline style.css:724]: - [ ] Nice work so far. However, the project image for the mobile pop-up is not displaying as shown below :point_down: . As a result of this it is difficult to identify the close button which is white, the same color as the background color. 
Kindly check and make sure that the image displays properly and the close button is also visible. :pray: 

- [ ] The margin around the mobile pop-up increases with increasing screen width which should not be so. Kindly ensure that the margins other dimensions of the pop-up window remains the same as the design for all screen sizes of the mobile version.

### Your design 

Screenshot from 2023-04-15 17-00-11



### Actual design 


Screenshot from 2023-04-15 16-59-31

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

this close2

## Highlights
Nice design :+1: 
Pop-up appears on button click. :+1: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @victormc13 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [ ] The previous reviewer pointed out that the margin around the mobile pop-up window increases as the screen width expands, which is not desirable. It is important to maintain consistent margins and dimensions for the pop-up window across all screen sizes in the mobile version, as outlined in the design. Please take the necessary steps to ensure that the margins and dimensions of the pop-up window remain consistent and do not vary with changes in screen size :+1:.

**Your design**

Screenshot from 2023-04-15 12-45-23


- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joel-kalema — APPROVED** [review]
Hi @victormc13 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**